docs/serial_recovery.md

```diff
@@ -0,0 +1,113 @@
+<!--
+  - SPDX-License-Identifier: Apache-2.0
+
+  - Copyright (c) 2022 Nordic Semiconductor ASA
+
+  - Original license:
+
+  - Licensed to the Apache Software Foundation (ASF) under one
+  - or more contributor license agreements.  See the NOTICE file
+  - distributed with this work for additional information
+  - regarding copyright ownership.  The ASF licenses this file
+  - to you under the Apache License, Version 2.0 (the
+  - "License"); you may not use this file except in compliance
+  - with the License.  You may obtain a copy of the License at
+
+  -  http://www.apache.org/licenses/LICENSE-2.0
+
+  - Unless required by applicable law or agreed to in writing,
+  - software distributed under the License is distributed on an
+  - "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  - KIND, either express or implied.  See the License for the
+  - specific language governing permissions and limitations
+  - under the License.
+-->
+
+# Serial Recovery
+
+Mcuboot implements a Simple Management Protocol (SMP) server. SMP is a basic transfer encoding for use with the MCUmgr management protocol.
+For more information about MCUmgr and SMP. Please refer [the zephyr-rtos device managemnt documentation](https://docs.zephyrproject.org/latest/services/device_mgmt/index.html#device-mgmt) for detail of these components.
+
+MCUboot supports only a subset of the MCUmgr commands:
+* echo
+* reset
+* image list
+* image upload
+
+It also might support system specific mcumgr commands which depends on given mcuboot-port.
+This extensioin is supposed to be enabled by ``MCUBOOT_PERUSER_MGMT_GROUP_ENABLED`` option.
```

Comment:
```suggestion

It also might support system specific MCUmgr commands which depends on the given mcuboot-port.
This extension can enabled by ``MCUBOOT_PERUSER_MGMT_GROUP_ENABLED`` option. ???
```
Marked the second line with ??? as this Kconfig does not exist, and the one in the previous file does not enable the commands themselves, it allows enabling the commands (which are opt-in not opt-out)

Reply:
right this is just mcuboot option which might be covered under zephyr-port Kconfig option.